src/app/ApplicationTemplate.Shared/Presentation/Welcome/OnboardingPageViewModel.cs

```diff
@@ -1,25 +1,60 @@
 ﻿using System;
+using System.Threading;
+using System.Threading.Tasks;
 using ApplicationTemplate.Business;
 using Chinook.DynamicMvvm;
 using Chinook.SectionsNavigation;
 using Chinook.StackNavigation;
+using Microsoft.Extensions.Localization;
+using ResourceLoader = Windows.ApplicationModel.Resources.ResourceLoader;
 
 namespace ApplicationTemplate.Presentation
 {
 	public partial class OnboardingPageViewModel : ViewModel
 	{
-		public IDynamicCommand NavigateToLogin => this.GetCommandFromTask(async ct =>
+		public OnboardingPageViewModel(bool isFromSettingsPage = false)
 		{
-			await this.GetService<ISectionsNavigator>().NavigateAndClear(ct, () => new LoginPageViewModel(isFirstLogin: true));
-			await this.GetService<IApplicationSettingsService>().CompleteOnboarding(ct);
+			this.IsFromSettingsPage = isFromSettingsPage;
+		}
+
+		public bool IsFromSettingsPage
```

Comment:
Can this property change value? It seems like it should not. When a property never changes, use a normal property instead of a dynamic property.